tests/stub/notification_filters/notification_filters_base.py

```diff
@@ -0,0 +1,95 @@
+from nutkit.frontend import Driver
+import nutkit.protocol as types
+from tests.shared import TestkitTestCase
+from tests.stub.shared import StubServer
+
+
+class NotificationFiltersBase(TestkitTestCase):
+    _auth = types.AuthorizationToken("basic", principal="neo4j",
+                                     credentials="pass")
+
+    def setUp(self):
+        super().setUp()
+        self._server = StubServer(9010)
+        self._uri = "bolt://%s" % self._server.address
+        self._driver = None
+
+    def tearDown(self):
+        self._server.reset()
+        if self._driver:
+            self._driver.close()
+        return super().tearDown()
+
+    def _new_driver(self, notifications):
+        return Driver(self._backend, self._uri, self._auth,
+                      notification_filters=notifications)
+
+    def _run_test_get_summary(self, filters, script_params,
+                              script="driver_notification_filters.script"):
+        self._server.start(self.script_path(script),
+                           vars_=script_params)
+        self._driver = self._new_driver(filters)
+        session = self._driver.session("w", database="neo4j")
+        cursor = session.run("CREATE (:node)")
+        result = cursor.consume()
+        self._server.done()
+        return result
+
+    @staticmethod
+    def configs():
+        return [
+            {
+                "filters": ["None"],
```

Comment:
```suggestion
                "filters": ["NONE"],
```
For consistency and in accordance with [the ADR](https://github.com/neo-technology/drivers-adr/pull/39/files#diff-ed02d11f49893acbb64270de326321d9bcc503f0a596befa14fb2672485e8308R65) (internal link).